models/articles.models.js

```diff
@@ -0,0 +1,30 @@
+const db = require("../db/connection");
+
+exports.fetchTopics = () => {
+  return db.query(`SELECT * FROM topics;`).then((data) => {
+    return data.rows;
+  });
+};
+
+exports.fetchArticleById = (articleId) => {
+  return db
+    .query(`SELECT * FROM articles WHERE article_id = $1`, [articleId])
+    .then((data) => {
+      return data.rows[0];
+    });
+};
+
+exports.updateArticleById = (articleId, inc_votes) => {
+  return db
+    .query(
+      `UPDATE articles SET votes = votes + $1 WHERE article_id = $2 RETURNING *;`,
+      [inc_votes, articleId]
+    )
+    .then((data) => {
+      console.log(data.rows);
```

Comment:
I reckon you won't need this one anymore... delete your console.logs as soon as you're done with them. It's a good habit to get into, helps keep your code neat and clean 👍🏼